app/templates/public.hbs

```diff
@@ -38,7 +38,7 @@
       <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="three wide column public">
+      <div class="{{if device.isMobile '' 'three wide column public'}}">
```

Comment:
Don't leave the class empty for a mobile screen, divs with no classes are a bad practice.